sdk/eventhub/azure-eventhubs/azure/eventhub/aio/client_async.py

```diff
@@ -85,33 +97,31 @@ def _create_auth(self, username=None, password=None):
                                                 get_jwt_token, http_proxy=http_proxy,
                                                 transport_type=transport_type)
 
+    async def _handle_exception(self, exception, retry_count, max_retries):
+        await _handle_exception(exception, retry_count, max_retries, self, log)
+
+    async def _close_connection(self):
+        self._conn_manager.reset_connection_if_broken()
```

Comment:
`reset_connection_if_broken` should be an async function

Reply:
yes, fixed